apps/api/internal/govcontext/handler_test.go

```
@@ -0,0 +1,73 @@
+	"strings"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/mosamlife/wpmgr/apps/api/internal/domain"
+)
+
+func bindCtx(body string) *gin.Context {
+	gin.SetMode(gin.TestMode)
+	c, _ := gin.CreateTestContext(httptest.NewRecorder())
+	c.Request = httptest.NewRequest("PATCH", "/api/v1/sites/x/context", strings.NewReader(body))
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)